build.sbt

```diff
@@ -743,7 +765,7 @@ addCommandAlias("validateKernelJS", "kernelLawsJS/test")
 addCommandAlias("validateFreeJS", "freeJS/test") //separated due to memory constraint on travis
 addCommandAlias("validate", ";clean;validateJS;validateKernelJS;validateFreeJS;validateJVM")
 
-addCommandAlias("prePR", ";fmt;++2.11.12;validateBC")
+addCommandAlias("prePR", ";fmt;++2.11.12 mimaReportBinaryIssues")
```

Comment:
unfortunately, sbt doesn't seem to work when trying to fix to a scala version when running a command alias.
I filed an issue https://github.com/sbt/sbt/issues/4599 
by this change we no longer run `binCompatTest/test`, but I think it's rare that it's accidentally broken before PR.